pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computeregionnetworkendpointgroup/privateserviceconnectioncomputeregionnetworkendpointgroup/create.yaml

```diff
@@ -17,11 +17,11 @@ kind: ComputeRegionNetworkEndpointGroup
 metadata:
   name: computeregionnetworkendpointgroup-${uniqueId}
   annotations:
-    cnrm.cloud.google.com/project-id: project-${uniqueId}
+    cnrm.cloud.google.com/project-id: ${TEST_DEPENDENT_ORG_PROJECT_ID}
 spec:
   region: us-west3
   networkEndpointType: PRIVATE_SERVICE_CONNECT
-  pscTargetService: https://www.googleapis.com/compute/v1/projects/project-${uniqueId}/regions/us-west3/serviceAttachments/computeserviceattachment-${uniqueId}
+  pscTargetService: "us-west3-cloudkms.googleapis.com"
```

Comment:
This looks like a real change; we are not targeting the ComputeServiceAttachment in our dependencies.yaml (?)

Reply:
I got "invalid value for field" error on this one. Checked the TF [sample](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_region_network_endpoint_group#example-usage---region-network-endpoint-group-psc), seems the field should be a string with format "asia-northeast3-cloudkms.googleapis.com", or reference to google_compute_service_attachment.

Reply:
Seems to be it's a backwards incompatible change made by the API (i.e. previously valid value is no longer valid). Can we file a ticket to the API team?

Reply:
Just for future reference, for similar incompatible change made by API team(and I believe that's very normal?), in addition to monitor failures and update our tests, shall we also file bugs to API team? Several concerns:
1. I feel they have nothing to do with our test failures?
2. KCC users might need to change the test value to their real value by following the updated field guidance, so that might not be broken changes to our customers?
3. What's the expected action item from us(KCC) and the API team(if we file a bug?)
Thanks for the suggestion!

Reply:
> 1. I feel they have nothing to do with our test failures?

If an API change breaks our test cases, there are chances customers will be exposed with this breaking change and file customer issues to us. Detecting such breaking changes is a positive side effect of our integration tests though it is not the main goal.

> 2. KCC users might need to change the test value to their real value by following the updated guidance, so that might not be broken changes to our customers?

Yes, it might not break our customers, but it is a [breaking change](go/yaqs/2661200769264386048) because customers who used to specify the value like `https://www.googleapis.com/compute/v1/projects/project-${uniqueId}/regions/us-west3/serviceAttachments/computeserviceattachment-${uniqueId}` and reconciled successfully will suddenly see unexpected failures.

> 3. What's the expected action item from us(KCC) and the API team(if we file a bug?)

Not really much action item from our side unless we have a customer issue.
Regarding the API team, it depends:
* Ideal outcome: API team reverts their breaking change (example: https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/521#issuecomment-912788195).
* Expected outcome: API team is aware an internal team has been impacted by their breaking change, and it is a starting point if future customer issues need to be addressed together. They also explains why this change is necessary/okay to do.
* Suboptimal outcome: API team ignores us.